src/modules/storage/LedgerStorage.ts

```diff
@@ -518,4 +518,31 @@ export class LedgerStorage extends Storages
 
         this.query(sql, [], onSuccess, onError);
     }
+
+    /**
+     * Returns the height of the block to be added next
+     */
+    public getExpectedBlockHeight(): Promise<number>
+    {
+        return new Promise<number>((resolve, reject) =>
+        {
+            let sql = `select height from blocks order by height desc limit 1;`;
```

Comment:
For readability, Looks good to use DML of SQL in capital letters.

Reply:
I changed it as below.
`SELECT height FROM blocks ORDER BY height DESC LIMIT 1;`